draft-ysl-cats-metric-definition.md

```diff
@@ -94,67 +90,72 @@ This document uses the following terms defined in {{!I-D.ietf-cats-framework}}:
 
 # Definition of Metrics
 
-Definition and usage of specific metrics are related to the intended use case. However, when considering disseminating compute metrics to network devices, appropriate categorization and abstraction of CATS metrics is required in order to avoid introducing extra complexity into the network.
-
-This document defines three abstract metric levels to meet different requirements use cases listed in {{!I-D.ietf-cats-usecases-requirements}}:
+Introducing a definition of metrics requires balancing the following trade-off: if the metrics are too fine-grained, they become unscalable due to the excessive number of metrics that must be communicated through the protocol. Conversely, if the metrics are too coarse-grained, they may lack the necessary information to make informed decisions. To ensure scalability while providing sufficient detail for effective decision-making, we propose a definition of metrics that incorporates three levels of abstraction:
```

Comment:
"through the protocol" I remember Med gave a comment previously, asking us to give an example on what protocol. So perhaps here we should add an example. "through the protocol(e.g., BGP)"

Reply:
Good point. I pushed this change: https://github.com/VMatrix1900/draft-cats-metric-definition/pull/8/commits/529c28a141cab00daffa213de8e5da8ff8ed2222

Let me know if this addresses your comment, thanks.